NumberValidator/Validators/NO/Fodselnummer.cs

```diff
@@ -0,0 +1,168 @@
+// fodselsnummer.py - functions for handling Norwegian birth numbers
+// coding: utf-8
+
+// Copyright (C) 2018 Ilya Vihtinsky
+// Copyright (C) 2018 Arthur de Jong
+// Copyright (C) 2020 Leon Sand�y
+// Copyright (C) 2021 Krishna Israni
+
+// This library is free software; you can redistribute it and/or
+// modify it under the terms of the GNU Lesser General Public
+// License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+
+// This library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+// Lesser General Public License for more details.
+
+// You should have received a copy of the GNU Lesser General Public
+// License along with this library; if not, write to the Free Software
+// Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA
+// 02110-1301 USA
+
+// F�dselsnummer (Norwegian birth number, the national identity number).
+// The F�dselsnummer is an eleven-digit number that is built up of the date of
+// birth of the person, a serial number and two check digits.
+// More information:
+// * https://no.wikipedia.org/wiki/F%C3%B8dselsnummer
+// * https://en.wikipedia.org/wiki/National_identification_number#Norway
+
+using System;
+using System.Globalization;
+using NumberValidator.Helpers;
+
+namespace NumberValidator.Validators.NO
+{
+    public class FSN : IValidator
+    {
+        public bool IsValid(string fsn)
+        {
+            try
+            {
+                Validate(fsn);
+                return true;
+            }
+            catch
+            {
+                return false;
+            }
+        }
+
+        public int CheckDigit1(string fsn)
+        {
+            var weights = new[] { 3, 7, 6, 1, 8, 9, 4, 5, 2 };
+
+            var sum = 0;
+            for (var index = 0; index < 9; index++)
+            {
+                sum += int.Parse(fsn[index].ToString()) * weights[index];
+            }
+            sum = 11 - (sum % 11);
+
+            return (sum);
+        }
+
+        public int CheckDigit2(string fsn)
+        {
+            var weights = new[] { 5, 4, 3, 2, 7, 6, 5, 4, 3, 2 };
+            var sum = 0;
+            for (var index = 0; index < 10; index++)
+            {
+                sum += int.Parse(fsn[index].ToString()) * weights[index];
+            }
+            sum = 11 - (sum % 11);
```

Comment:
I would prefer assignment to variable `sum` in a new line. Show a separation from for loop.